site/docs/v1/tech/guides/_recovery-codes-blurb.adoc

```diff
@@ -0,0 +1,14 @@
+When you first add an MFA method to a user, the response is a set of codes:
+
+[source,json]
+.Recovery Codes Response JSON
+----
+include::docs/src/json/two-factor/response-with-recovery-codes.json[]
```

Comment:
This is breaking for me.  
- URL: `https://site-local.fusionauth.io/docs/v1/tech/guides/multi-factor-authentication/#recovery-codes`

<img width="1066" alt="Screen Shot 2021-04-29 at 11 10 47 AM" src="https://user-images.githubusercontent.com/16090626/116591016-9f5e6580-a8db-11eb-89b8-cec20244cacd.png">

Reply:
Thanks!